ceph_iscsi_config/target.py

```diff
@@ -177,25 +177,25 @@ def enable_active_tpg(self, config):
         :return: None
         """
 
+        index = 0
```

Comment:
I think code wise the patch looks ok.

ceph-iscsi and tcmu-runner though assumed that there would only be one iscsi portal per target port group. If we hit this loop more than once then I think bind_alua_group_to_lun is going to fail because we pass in the same tpg tag to ALUATargetPortGroup and it will try to do a create for the same tpg twice.

I think failing might be ok though, because the user would get an error while trying to set it up and so we would not allow it.

@ricardoasmarques you guys do not use target_core_user/tcmu-runner in active/passive mode right? So it should be ok for both of our types of set ups I think?

Reply:
@mikechristie Using multiple IPs in `lrbd` will create multiple portals in the same `tpg`.

For instance, in `ceph-iscsi` you will have:

```
tpg2
  - portals
    192.168.100.202:3160
tpg3
  - portals
    192.168.121.123:3160
```

and in lrbd you will have:

```
tpg2
  - portals
    192.168.100.202:3160
    192.168.121.123:3160
```

Isn't this equivalent? Should we change `ceph-iscsi` to create multiple portals in the same tpg?

Reply:
I think multiple portals in the same tpg will work for what we support in RHCS. I am doing some testing now.